### ISSUE TITLE
Create post-install script to automatically generate security key and add it to .env.example if necessary (i.e. initial project setup)

### ISSUE THREAD

**Issue by vidurb** [miranj/craft#5]
I think the title suffices for explanation. It'd be simple enough to implement and would ensure that the first time any project is created a corresponding security key is added to the repo so that the next person to clone does not need to set it up.

**Comment by rungta** [comment]
Good idea. However, Craft already does this for us as part of its setup script. I think it is this one https://github.com/miranj/craft/blob/master/composer.json#L45

So our script will simply need to copy the value from the freshly created `.env` and update that in `.env.example` as well.
